models/yang/sonic/sonic-tam.yang

```diff
@@ -52,6 +52,12 @@ module sonic-tam {
                 }
 
                 leaf ipaddress-type {
+                    must "(contains(../ipaddress, ':') and current()='ipv6') or
+                          (contains(../ipaddress, '.') and current()='ipv4')" {
+                        error-message "%Error: Wrong ipaddress for the specified ipaddress-type";
+                        error-app-tag "%Error: Wrong ipaddress for the specified ipaddress-type";
```

Comment:
This is incorrect error-app-tag format . Please refer to https://tools.ietf.org/html/rfc7950. 
Should be like :
error-app-tag invalid-ipaddress.

Reply:
Done